cobra/core/Model.py

```diff
@@ -552,6 +552,7 @@ def objective(self):
         return self.objective_reactions
 
     @objective.setter
+    @resettable
```

Comment:
No not a problem but still tinkering with objective_reactions in #324 so might as well leave it like this for now.